src/main/java/com/urlparser/entity/LinksFromUrl.java

```diff
@@ -0,0 +1,54 @@
+package com.urlparser.entity;
+
+import com.urlparser.model.Links;
+
+import org.jsoup.Jsoup;
+import org.jsoup.nodes.Document;
+import org.jsoup.nodes.Element;
+import org.jsoup.select.Elements;
+import org.apache.log4j.LogManager;
+import org.apache.log4j.Logger;
+
+
+import java.io.IOException;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+
+
+public class LinksFromUrl implements Links {
+
+  private String url;
```

Comment:
@undsett it is a good practice to make such attributes `final` so we are saying we wont change this field after object created - this is immutable object